pkg/systemd/services/timer-dialog.jsx

```diff
@@ -84,6 +87,15 @@ const CreateTimerDialogBody = ({ owner }) => {
         validationFailed.command = true;
     if (!/^[0-9]+$/.test(delayNumber))
         validationFailed.delayNumber = true;
+    if (custom) {
+        cockpit.spawn(["systemd-analyze", "calendar", custom])
+                .then(() => {
+                    setValidateCustom(false);
+                })
+                .catch(() => {
+                    setValidateCustom(true);
+                });
+    }
```

Comment:
This could be written just like:
```
        cockpit.spawn(["systemd-analyze", "calendar", custom])                  
                .then(() => setValidateCustom(false))                           
                .catch(() => setValidateCustom(true));          
```

Reply:
I cleaned this up.  Thanks for the suggestion.